extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeChannelExtractorTest.java

```diff
@@ -518,9 +518,8 @@ public void testFeedUrl() throws Exception {
 
         @Test
         public void testSubscriberCount() throws Exception {
-            // there is no "Subscribe" button
             long subscribers = extractor.getSubscriberCount();
-            assertEquals("Wrong subscriber count", -1, subscribers);
+            assertEquals("Wrong subscriber count", 0, subscribers);
```

Comment:
This test is now actually wrong, since the subscriber count should be `ITEM_COUNT_UNKNOWN` = `-1`, not `0`, when the subscriber count is hidden, otherwise NewPipe wouldn't know it should hide the subscriber count